bundles/org.eclipse.swt/Eclipse SWT/cocoa/org/eclipse/swt/widgets/Display.java

```diff
@@ -5277,7 +5277,6 @@ public void syncExec (Runnable runnable) {
  */
 public <T, E extends Exception> T syncCall(SwtCallable<T, E> callable) throws E {
 	Objects.nonNull(callable);
-	@SuppressWarnings("unchecked")
 	T[] t = (T[]) new Object[1];
```

Comment:
```suggestion
	Object[] t = new Object[1];
```

and cat the return value as with the exception?

Reply:
Obsolete with applying Heiko's suggestion and using AtomicReference.